kodilla-hibernate/src/main/java/com/kodilla/hibernate/invoice/Item.java

```diff
@@ -0,0 +1,88 @@
+package com.kodilla.hibernate.invoice;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "ITEMS")
+public class Item {
+    private int id;
+    private BigDecimal price;
+    private int quantity;
+    private BigDecimal value;
+
+    private Product product;
+    private Invoice invoice;
```

Comment:
Ogólnie to `Item` nie powinien mieć w sobie wciągniętej encji `Invoice` patrząc na treść zadania. Nie jest Ci ona tu potrzebna raczej